dll/win32/rasdlg/lang/ro-RO.rc

```diff
@@ -106,50 +106,50 @@ BEGIN
     AUTOCHECKBOX "&Aplică regulile de formare a numerelor", 1048, 20, 81, 219, 10, BS_TOP | BS_MULTILINE
     GROUPBOX "Co&mentariu", 1052, 7, 102, 243, 36
     EDITTEXT 1050, 17, 115, 224, 14, ES_AUTOHSCROLL
-    DEFPUSHBUTTON "Con&firmă", 1, 127, 147, 60, 14
-    PUSHBUTTON "A&nulează", 2, 191, 147, 60, 14
+    DEFPUSHBUTTON "&OK", 1, 127, 147, 60, 14
+    PUSHBUTTON "&Revocare", 2, 191, 147, 60, 14
```

Comment:
This change introduces an accelerator collision with the line
 LTEXT "Codul &regiunii:", 1055, 18, 17, 54, 8
Please check your romanian Windows to find out which of both lines need to be changed.